apps/native-component-list/src/screens/FileSystemScreen.tsx

```diff
@@ -335,6 +352,7 @@ export default class FileSystemScreen extends React.Component<object, State> {
         <ListButton onPress={this._readAsset} title="Read Asset" />
         <ListButton onPress={this._getInfoAsset} title="Get Info Asset" />
         <ListButton onPress={this._copyAndReadAsset} title="Copy and Read Asset" />
+        <ListButton onPress={this._createUTF8Uri} title="Create UTF-8 uri" />
```

Comment:
would recommend adding the test case in **apps/test-suite/tests/FileSystem.js** rather than NCL if feasible. that would save a little attention cost when doing QA.

Reply:
Sure, I've just moved it to apps/test-suite/tests/FileSystem.js